exporter.go

```diff
@@ -268,36 +284,62 @@ func initWbem() {
 
 func main() {
 	var (
-		listenAddress = kingpin.Flag(
+		app         = kingpin.New("windows_exporter", "")
+		configFiles = app.Flag(
+			"config.files",
+			"Comma-separated list of YAML configuration file(s) to use. Values set in these file(s) will override flags.",
```

Comment:
Agreed on the priority, I'll amend this to set the flags as higher priority.
Would support for multiple configurations files be desirable, or would a single config file be acceptable initially?

Reply:
I think a single file is a good start to iron out how we want the basic things to work and lets us avoid a lot of these tricky questions for now :) I'd also guess that it would be sufficient for most users.